shell/client/admin/user-details.html

```diff
@@ -285,6 +285,36 @@
     <h2>Identities</h2>
     {{> newAdminUserDetailsIdentityTable account=account }}
 
+    <h2>Status</h2>
+
+    <div style="display: flex; flex-direction: row;">
+      Active
+    </div>
+    <form class="standard-form">
+      <div class="button-row">
+        <button type="button" name="delete-account">Delete...</button>
+      </div>
+    </form>
+
+    {{#if showDeletePopup}}
+      {{#modalDialogWithBackdrop onDismiss=cancelDelete}}
+        {{#if deleteError}}
+          {{#focusingErrorBox}}
+            {{deleteError}}
+          {{/focusingErrorBox}}
+        {{/if}}
+
+        <p>Are you sure? This operation is permanent and irreversible.</p>
+        <form class="standard-form">
+          <div class="button-row">
+            <button class="danger" type="button"
```

Comment:
Hmmm, this isn't showing up in the alarming-red I was expecting, which is quite likely my fault.

Mind adding the following patch to the standard-form in `shell/client/styles/_partials.scss`?

```diff
diff --git a/shell/client/styles/_partials.scss b/shell/client/styles/_partials.scss
index 9b20e73..5aec645 100644
--- a/shell/client/styles/_partials.scss
+++ b/shell/client/styles/_partials.scss
@@ -194,6 +194,9 @@
   button[type=button] {
     @extend %button-base;
     @extend %button-secondary;
+    &.danger {
+      @extend %button-danger;
+    }
   }
 
   &.disabled {
```